code/_helpers/lists.dm

```diff
@@ -760,3 +760,47 @@ proc/dd_sortedTextList(list/incoming)
 #define LAZYACCESS(L, I) (L ? (isnum(I) ? (I > 0 && I <= L.len ? L[I] : null) : L[I]) : null)
 #define LAZYLEN(L) length(L)
 #define LAZYCLEARLIST(L) if(L) L.Cut()
+
+/*
+Two lists may be different (A!=B) even if they have the same elements.
+This actually tests if they have the same entries and values.
+*/
+/proc/same_entries(var/list/first, var/list/second)
```

Comment:
Isn't that duplicate `~= operator` functionality?

Reply:
This doesnt work for list, i tried to compare lists many times before. They can only be compared by their values